components/Chips/src/MDCChipView.m

```diff
@@ -568,6 +570,13 @@ - (void)updateState {
   [self updateTitleColor];
 }
 
+#pragma mark - Custom touch handling
+
+- (BOOL)pointInside:(CGPoint)point withEvent:(__unused UIEvent *)event {
+  CGRect hitAreaRect = UIEdgeInsetsInsetRect(CGRectStandardize(self.bounds), self.hitAreaInsets);
```

Comment:
is there a reason we need `CGRectStandardize`? will the values of the bounds itself be negative?

Reply:
Bounds can be negative, especially with Autolayout. The general recommendation on the team has been to standardize `bounds` and `rect` before manipulating them. Since `UIEdgeInsetsInsetRect` will not standardize a CGRect first, it's best to standardize before applying the insets.